APCAppCore/APCAppCore/Library/Scheduler/APCScheduler.h

```diff
@@ -123,60 +119,42 @@ typedef void (^APCSchedulerCallbackForFetchingCount) (NSUInteger count, NSError
                       usingQueue: (NSOperationQueue *) queue
                  toReportResults: (APCSchedulerCallbackForTaskGroupQueries) callbackBlock;
 
-- (NSArray *) querySchedulesActiveOnDayOfDate: (NSDate *) date
-                                   fromSource: (APCScheduleSource) scheduleSource
-                                    inContext: (NSManagedObjectContext *) context
-                               returningError: (NSError * __autoreleasing *) errorToReturn;
-
 
 
 // ---------------------------------------------------------
 #pragma mark - Importing
 // ---------------------------------------------------------
 
-- (void) fetchTasksAndSchedulesFromServerAndThenUseThisQueue: (NSOperationQueue *) queue
-                                            toDoThisWhenDone: (APCSchedulerCallbackForFetchAndLoadOperations) callbackBlock;
-
-- (void) loadTasksAndSchedulesFromDiskAndThenUseThisQueue: (NSOperationQueue *) queue
-                                         toDoThisWhenDone: (APCSchedulerCallbackForFetchAndLoadOperations) callbackBlock;
-
-- (void) loadTasksAndSchedulesFromDiskFileNamed: (NSString *) fileNameWithExtension
-                                       inBundle: (NSBundle *) bundle
-                                assigningSource: (APCScheduleSource) scheduleSource
-                            andThenUseThisQueue: (NSOperationQueue *) queue
-                               toDoThisWhenDone: (APCSchedulerCallbackForFetchAndLoadOperations) callbackBlock;
-
-- (void) importScheduleFromDictionary: (NSDictionary *) scheduleContainingTasks
-                      assigningSource: (APCScheduleSource) scheduleSource
-                  andThenUseThisQueue: (NSOperationQueue *) queue
-                     toDoThisWhenDone: (APCSchedulerCallbackForFetchAndLoadOperations) callbackBlock;
-
-
+- (void) fetchTasksFromServerAndThenUseThisQueue: (NSOperationQueue *) queue
+                                toDoThisWhenDone: (APCSchedulerCallbackForFetchAndLoadOperations) callbackBlock;
 
 // ---------------------------------------------------------
-#pragma mark - Converting PotentialTasks to ScheduledTasks
+#pragma mark - MANAGING STARTING AND FINISHING OF TASKS
 // ---------------------------------------------------------
 
 /**
- We only save ScheduledTasks when the user actually does
- something with them.  The -fetch methods above return
- (among other things) PotentialTask objects which show when
- a user *could* perform a task.  This method lets us
- convert from one of those PotentialTasks to a
- ScheduledTask, representing the user's real work.  Please
- be sure to call -deleteScheduledTask: if the user cancels
- that operation.
+ Starts the current task which involves setting a start date to
+ the current time, saving the started task to CoreDate, and purging
```

Comment:
Also do these methods (also) notify Bridge Server via the BridgeSDK when a task is started/finished?